returns/functions.py

```diff
@@ -167,3 +170,20 @@ def not_(function: Callable[..., bool]) -> Callable[..., bool]:
     def wrapped_function(*args, **kwargs) -> bool:  # noqa: WPS430
         return not function(*args, **kwargs)
     return wrapped_function
+
+
+def is_io(container: BaseContainer) -> bool:
```

Comment:
Is it possible to move it to `pipeline.py`? I am asking because I want to get rid of this nested import.

Reply:
Nope, we should add a new module for it.
We would add `is_context, `is_result`, and `is_future`.

Let's call it `instances.py` maybe?

Reply:
What about `containers.py`? Because all of those functions will do something with containers!

Reply:
@sobolevn ?

Reply:
Containers is too general. We check if something is an `instance` of `io`, `future`, etc. So, `instances.py` looks better for me.

Reply:
`instances` is too general, as well, this topic is much abstract kkkkk. But it's, I'll use `instances.py`!